GUI/src/main/kotlin/ui/components/selectVideoScreen/LoadingDialog.kt

```diff
@@ -0,0 +1,43 @@
+package ui.components.selectVideoScreen
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.AlertDialog
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material3.Button
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun LoadingDialog(onCancel: () -> Unit = {}) {
+    AlertDialog(
```

Comment:
The other AlertDialogs we use are now styled like this:

![image](https://github.com/amosproj/amos2023ws03-gui-frame-diff/assets/52202194/0714ef02-c6d9-4832-ad2d-f53993da061e)

You can either change that here or is will be done in the design branch after rebasing on this one. If requested I can also change this here.